doc/book/router.aura.md

```diff
@@ -0,0 +1,172 @@
+# Advanced Aura.Router usage
+
+[Aura.Router](https://github.com/auraphp/Aura.Router) provides a plethora of
+methods for further configuring the router instance. One of the more useful
+configuration is to provide default specifications:
+
+- A regular expression that applies the same for a given routing match:
+
+  ```php
+  // Parameters named "id" will only match digits by default:
+  $router->addTokens([
+    'id' => '\d+',
+  ]);
+  ```
+
+- A default parameter and/or its default value to always provide:
+
+  ```php
+  // mediatype defaults to "application/xhtml+xml" and will be available in all
+  // requests:
+  $router->addValues([
+    'mediatype' => 'application/xhtml+xml',
+  ]);
+  ```
+
+- Only match if secure (i.e., under HTTPS):
+
+  ```php
+  $router->setSecure(true);
+  ```
+
+In order to specify these, you need access to the underlying Aura.Router
+instance, however, and the `RouterInterface` does not provide an accessor!
+
+The answer, then, is to use dependency injection. This can be done in two ways:
+programmatically, or via a factory to use in conjunction with your container
+instance.
+
+## Programmtic Aura.Router Creation
+
+To handle it programmatically, you will need to setup the Aura.Router instance
+manually, inject it into a `Zend\Expressive\Router\Aura` instance, and inject
+use that when creating your `Application` instance.
+
+```php
+<?php
+use Aura\Router\RouterFactory;
+use Zend\Expressive\AppFactory;
+use Zend\Expressive\Router\Aura as AuraBridge;
+
+$auraRouter = (new RouterFactory())->newInstance();
+$auraRouter->setSecure(true);
+$auraRouter->addValues([
```

Comment:
Pretty sure on both: https://github.com/auraphp/Aura.Router#default-route-specifications details using these on the `$router` instance to set defaults.